src/main/java/emu/grasscutter/command/commands/HelpCommand.java

```diff
@@ -68,6 +68,13 @@ void SendAllHelpMessage(Player player, List<Command> annotations) {
                         builder.append(alias).append(" ");
                     }
                 }
+                builder.append("\n").append(translate(player, "commands.help.permission"));
```

Comment:
This is good, it would also be useful for the single command case at line 49:

```java
if (player != null && !Objects.equals(annotation.permission(), "") && !player.getAccount().hasPermission(annotation.permission())) {
    builder.append("\n Warning: You do not have permission to run this command.");
}
```
to
```java
builder.append("\n").append(translate(player, "commands.help.permission"));
if(annotation.permission().isEmpty() || annotation.permission().isBlank()) {
    builder.append(translate(player, "commands.help.no_permission"));
} else {
    builder.append(annotation.permission());
    if (player != null && !player.getAccount().hasPermission(annotation.permission())) {
        builder.append("\n Warning: You do not have permission to run this command.");  // Should actually be changed to a translation key
    }
}
```